lib/java/com/google/android/material/floatingactionbutton/res/color/mtrl_fab_bg_color_selector.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Since these are the same as EFAB and I expect them to behave the same way, maybe both classes should use the same selectors?

Reply:
I was thinking the same. For the naming - would it be fine if the color resources used the `mtrl_fab` prefix for both FAB and EFAB?

Reply:
Done @cketcham ✅